lib/tasks/evm_dba.rake

```diff
@@ -193,6 +193,53 @@ namespace :evm do
       end
     end
 
+    namespace :dump do
+      require Rails.root.join("lib", "evm_database_ops").expand_path.to_s
+      desc 'Dump the local ManageIQ EVM Database (VMDB) to a local file'
+      task :local do
+        require 'trollop'
+        opts = Trollop.options(EvmRakeHelper.extract_command_options) do
+          opt :local_file, "Destination file", :type => :string, :required => true
+          opt :username,   "Username",         :type => :string
+          opt :password,   "Password",         :type => :string
+          opt :hostname,   "Hostname",         :type => :string
+          opt :dbname,     "Database name",    :type => :string
+        end
+
+        opts.delete_if { |_, v| v.nil? }
+        EvmDatabaseOps.dump(opts)
+
+        exit # exit so that parameters to the first rake task are not run as rake tasks
+      end
+
+      desc 'Dump the local ManageIQ EVM Database (VMDB) to a remote file'
+      task :remote do
```

Comment:
Do we need the restore for the dump side of things?

Reply:
Nope, this is handled in `PostgresAdmin`:

https://github.com/ManageIQ/manageiq-gems-pending/blob/559540c54a32f202a60db14d6ba4a51bf01262e5/lib/gems/pending/util/postgres_admin.rb#L93-L102

Oddly enough, written by you:  https://github.com/ManageIQ/manageiq-gems-pending/commit/dbb78da63c34a01f4a5be0a4c53fe6f6830a7573

Reply:
Lol sorry, I read this as `task :restore do`
:man_facepalming:

Reply:
To be more clear, I was suggesting removing this task because I thought it was restoring a pg_dump which I don't really want to do unless it was an old backup (from before we removed it as a backup option).

Reply:
That is fair.  Yeah, I really don't plan to support `pg_dump` restores myself.  I have a fun set of directions for importing DBs when working on performance issues and such, so me personally, I only really care about getting the `pg_dump` stuff working from a "backup" perspective.

The fact that it "can work" in the `:restore` tasks is just a nice plus.